lib/Zonemaster/Backend/Translator.pm

```diff
@@ -15,6 +17,19 @@ require Zonemaster::Engine::Logger::Entry;
 
 extends 'Zonemaster::Engine::Translator';
 
+Readonly my %TAG_DESCRIPTIONS => (
+    UNABLE_TO_FINISH_TEST => sub {
+        __x    # BACKEND_TEST_AGENT:UNABLE_TO_FINISH_TEST
+          "The test was unabled to finish.", @_;
```

Comment:
There is alread the SYSTEM level. Is there any problem to add that message to that level? If not, then we do not have to create a new level.

Reply:
Maybe reuse SYSTEM yes, and so then we can be extra explicit for the message tag (if we want to) since it will not be displayed to users, e.g. "BACKEND_TEST_AGENT_UNABLE_TO_FINISH_TEST". I'm suggesting to be extra explicit because I think SYSTEM messages come from Engine exclusively.

Reply:
I think it is fine to have "ugly" tags (there are more of them) since they are usually not shown.

Reply:
I think we can live without migrating the database. I only means that those old tests, which probably are few and have short value, will be displayed differently. I think that is a better alternative than living with that strange pseudotag.